src/gc.c

```diff
@@ -2836,11 +2837,11 @@ void *jl_gc_perm_alloc_nolock(size_t sz, int zero)
     if (__unlikely(sz > gc_perm_size)) {
 #ifdef _OS_WINDOWS_
         void *pool = VirtualAlloc(NULL,
-                                  GC_PERM_POOL_SIZE + JL_SMALL_BYTE_ALIGNMENT,
+                                  GC_PERM_POOL_SIZE + JL_HEAP_ALIGNMENT,
```

Comment:
..... The comment above applys to these too....... The permgen allocation shouldn't be changed.